src/backend/src/services/ws/index.js

```diff
@@ -0,0 +1,42 @@
+export const Events = {
+  CREATE: 'create',
+  JOIN: 'join',
+};
+
+export function uuidv4() {
```

Comment:
Use uuid package instead of rolling our own implementation.

Reference: https://github.com/Tamas-Leung/CodeChamp/blob/main/src/backend/src/services/judge/language-generators.js#L1